### PR TITLE
Seeders for users and projects

### PR DESCRIPTION
**Pull request by brec9824**
@uidaho/lambda, review

The database can now be seeded with php artisan db:seed. Model factory has functions for creating just a specific class while some create its class plus its connected classes. Multiple seeders were added that contain multiple call options. Either self contained or non self contained.

### CODE REVIEW

File: database/seeds/ProjectsTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            factory(Project::class, 1)->create();
+            $newestID = DB::table('projects')->orderBy('created_at', 'desc')->first()->id;                  //Get the project that was just created's id
+            $imageURL = $faker->imageUrl(640, 480, 'technics');                                             //Get a new image
+            $data = file_get_contents($imageURL);
+            $new = base_path() . '/public/images/projects/' . 'product' . $newestID . '.jpg';
+            file_put_contents($new, $data);
+        }
+
+        //Creates projects and connected users
+        /*for ($i = 0; $i < 1; $i++)
```
Comment (rickbau5) [inline database/seeds/ProjectsTableSeeder.php]: this can be taken out?

Comment (brec9824) [inline database/seeds/ProjectsTableSeeder.php]: @rickbau5 They each call different model factory functions. One just creates the project and the other creates the project and the connected user inside of it. I couldnt find a better way then just commenting out the one not in use.

Comment (rickbau5) [inline database/seeds/ProjectsTableSeeder.php]: i meant the commented stuff

Comment (brec9824) [inline database/seeds/ProjectsTableSeeder.php]: @rickbau5 Ok fixed

**Comment by brandonjank** [comment]
Rebased and passing. Merging and deleting branch.

**Pull request merged**